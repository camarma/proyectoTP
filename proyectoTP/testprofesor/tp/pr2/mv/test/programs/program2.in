Push 65
Add
Push 2
out
end
