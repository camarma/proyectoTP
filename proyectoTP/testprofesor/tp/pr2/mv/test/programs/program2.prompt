step 15
step
quit