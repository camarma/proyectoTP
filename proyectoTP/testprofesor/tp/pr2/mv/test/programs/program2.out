Introduce el programa fuente
> > > > > El programa introducido es:
0: PUSH 65
1: ADD
2: PUSH 2
3: OUT
> Comienza la ejecucion de  PUSH 65
El estado de la maquina tras ejecutar la instruccion es:
Memoria: <vacia>
Pila de operandos: 65
Comienza la ejecucion de  ADD
Error en la ejecucion de la instruccion
> Comienza la ejecucion de  ADD
Error en la ejecucion de la instruccion
> 